app/views/liquids/_runkit.html.erb

```diff
@@ -2,7 +2,7 @@
   <code style="display: none">
     <%= preamble %>
   </code>
-  <code>
+  <code style="display: none;">
```

Comment:
@ledestin, I added this to avoid showing the code block before the iframe loads.

![May-11-2020 11-53-00](https://user-images.githubusercontent.com/833231/81582399-03f87b80-937e-11ea-8ce2-abb799f5ec84.gif)